content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

```diff
@@ -510,116 +524,56 @@ The SNI proxy will forward the traffic to port `443`.
     metadata:
       name: forward-downstream-sni
     spec:
-      filters:
-      - listenerMatch:
-          portNumber: 443
-          listenerType: SIDECAR_OUTBOUND
-        filterName: forward_downstream_sni
-        filterType: NETWORK
-        filterConfig: {}
+      configPatches:
+      - applyTo: NETWORK_FILTER
+        match:
+          context: SIDECAR_OUTBOUND
+          listener:
+            portNumber: 443
+            filterChain:
+              filter:
+                name: mixer
+        patch:
+          operation: INSERT_BEFORE
+          value:
+             name: forward-downstream-sni
```

Comment:
The name of the filter must be `forward_downstream_sni`, the name of the filter appears here https://github.com/istio/proxy/blob/a7ef110ba4e2b0f593c1a5c29e677da91e62014f/src/envoy/tcp/forward_downstream_sni/config.h#L36